api/src/main/java/org/openmrs/module/legacyui/api/LegacyUIService.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.legacyui.api;
+
+import java.util.Date;
+
+import org.openmrs.Concept;
+import org.openmrs.Patient;
+import org.openmrs.Provider;
+import org.openmrs.User;
+import org.openmrs.annotation.Authorized;
+import org.openmrs.api.APIException;
+import org.openmrs.api.OpenmrsService;
+import org.openmrs.util.PrivilegeConstants;
+
+/**
+ * Contains methods pertaining to Patients in the system
+ * 
+ * <pre>
+ * Usage:
+ * List&lt;Patient&gt; patients = Context.getPatientService().getAllPatients();
```

Comment:
Is this an effect of a copy and paste?

Reply:
[Fixed](https://github.com/openmrs/openmrs-module-legacyui/pull/128/commits/4753ae55d9f61c8dcc882fa4bb22bf60ec775b77)